### DISCUSSION TITLE
how to use the output of the multi retrieval QA chain?

### DISCUSSION THREAD

**Discussion by micuentadecasa** [FlowiseAI/Flowise#1796] [Q&A] [answered]
Hi, I'm using a multi retrieval QA chain, and it is working right, now I want to get the output of this chain as "context" for another llm chain, but I don't know how to do it, I will appreciate if you can help me.

related to it, I have an idea, I miss some info to know what component can be connected to, I mean can be a good idea to add in the info button of each component a kind of help with the possible components that the output can be connected to, or even the input.

**Comment by Jaredude** [comment, marked as answer]
This is currently not possible to link the output of a QA chain to an LLM chain. One workaround would be to call QA chain from within a custom tool and then link that custom tool to your LLM chain.

**Comment by micuentadecasa** [comment]
Thanks @Jaredude Jaredude.

Regards.

**Comment by code4rk-ai** [comment]
Can you suggest how to you custom tool
